src/javascripts/modules/ExpandListItem.js

```diff
@@ -0,0 +1,19 @@
+export default class ExpandListItem {
+  constructor(el) {
+    this.el = el
+    this.tag = el.parentNode
+    this.el.addEventListener('click', this.expand)
+  }
+
+  expand = () => {
+    let pressed = this.el.getAttribute('aria-pressed') === 'true'
+    if (this.tag.style.visibility === 'visible') {
+      this.tag.style.visibility = 'hidden'
+      this.el.setAttribute('aria-pressed', String('false'))
+    } else {
+      this.tag.style.visibility = 'visible'
+      this.el.setAttribute('aria-pressed', String(!pressed))
+    }
```

Comment:
I have some ideas for refactoring here. We can chat it in person, but it pertains to the following:

- Using tern ternaries instead of an if/else block
- Refactoring to use `aria-expanded` with `aria-controls` instead of `aria-pressed` and targeting the LI
- Using the `hidden` attribute (which is settable via true/false in JS) instead of css visibility